web-console/monitoring/alerts/alerts.md

```diff
@@ -0,0 +1,15 @@
+# Alerts
+
+## Alert Metrics
+
+![Alert details view with metrics](img/alert-metrics.png)
+- Initially the chart should be zoomed to a time period that at least includes the `for` attribute of the alert, but preferably 3-5 times that period as loading times allow
+- An orange line with an alert icon (if possible) shows the firing threshold for the alert
+- Clicking on the `View in Metrics` link takes the user to the metrics page and pre-populated with the relevant query for the alert
```

Comment:
This is a link to the Prometheus UI, correct? Could you update the text to state that?